### PR TITLE
Update README.md and Postinstall

### PR DESCRIPTION
**Pull request by bleachc**
I ran into a few problems trying to get this going on my machine. Mainly the Visual Studio C++ tools. Also the copy commands in the README.md right now are backwards and are escaping. So I updated that and also made a shell script to do it quickly in case I need to nuke node_modules.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,9 +80,9 @@ yarn install
 ```
 3. Copy iohook binaries
 ```sh
-mkdir -p node_modules\iohook\builds\electron-v80-win32-x64\build\Release\
-cp iohook\iohook.node node_modules\iohook\builds\electron-v80-win32-x64\build\Release\
-cp iohook\uiohook.dll node_modules\iohook\builds\electron-v80-win32-x64\build\Release\
+mkdir -p node_modules/iohook/builds/electron-v80-win32-x64/build/Release/
```
Comment (Ewpratten) [inline README.md:83]: Since you added a `copy-io.sh` script, you might as well replace these lines with `sh ./copy-io.sh`

Comment (bleachc) [reply README.md]: This is true. Should have done that. Thanks!

**Comment by DispatchCommit** [comment]
> Also the copy commands in the README.md right now are backwards and are escaping.

Yea that gave me issues as well. I was thinking of adding this to the post-install with yarn as well so it doesn't require an extra step.

**Comment by bleachc** [comment]
> 
> 
> > Also the copy commands in the README.md right now are backwards and are escaping.
> 
> Yea that gave me issues as well. I was thinking of adding this to the post-install with yarn as well so it doesn't require an extra step.

I thought about it. I haven't dealt too much with postinstall scripts before. So my thought was using a shell script might not be viable because not everyone has sh/bash. So what probably has to happen, unless someone else knows a better solution is making a postinstall JavaScript file.

**Comment by cinderblock** [comment]
#263 makes this irrelevant

**Pull request closed without merging**